rd-net/RdFramework/Impl/RdProperty.cs

```diff
@@ -155,7 +155,7 @@ public override void OnWireReceived(UnsafeReader reader)
 
       var rejected = IsMaster && version < myMasterVersion;
         
-      ReceiveTrace?.Log($"{this} :: oldver = {myMasterVersion}, newver = {version}, value = {value.PrintToString()}{(rejected ? " REJECTED" : "")}");
+      ReceiveTrace?.Log($"{this} :: oldver = {myMasterVersion}, newver = {version}, value = {value?.PrintToString()}{(rejected ? " REJECTED" : "")}");
```

Comment:
_(Same as above)_ `PrintToString` is supposed to handle null better (by printing `"<null>"`), so could we leave this as is?